src/Dialect/ONNX/ONNXOps/Math/ElementwiseBroadcast.cpp

```diff
@@ -161,6 +161,20 @@ LogicalResult ONNXBitShiftOp::inferShapes(
   return inferShapeForBroadcastingOps<ONNXBitShiftOp>(*this);
 }
 
+//===----------------------------------------------------------------------===//
+// CastLikeOp
+//===----------------------------------------------------------------------===//
+LogicalResult ONNXCastLikeOp::inferShapes(
```

Comment:
@chentong319 We previously talked about mapping the ONNXCastLike to use ONNXCast if it is possible in my next PR. Will create an issue to tag said progress :)

Reply:
If its removed before even hitting shape inference, we don't need this code at all

Reply:
I removed it